src/core/projectinfo.cpp

```diff
@@ -743,3 +744,39 @@ QVariantMap ProjectInfo::getCopyrightDecorationConfiguration()
 
   return configuration;
 }
+
+QVariantMap ProjectInfo::getImageDecorationConfiguration()
+{
+  QVariantMap configuration;
+  const QString configurationName = QStringLiteral( "Image" );
+
+  if ( QgsProject::instance()->readBoolEntry( configurationName, QStringLiteral( "/Enabled" ), false ) )
+  {
+    QString imagePath = QgsProject::instance()->readEntry( configurationName, QStringLiteral( "/ImagePath" ), QString() );
+    const QString resolvedPath = QgsSymbolLayerUtils::svgSymbolNameToPath( imagePath, QgsProject::instance()->pathResolver() );
+    const QFileInfo fileInfo( resolvedPath );
+    if ( fileInfo.exists() )
+    {
+      imagePath = resolvedPath;
+    }
+    else
+    {
+      imagePath = QStringLiteral( ":/images/qfield_logo.svg" );
```

Comment:
Does it mean that if the path is misconfigured or the file did not get to the phone we just show a qfield logo?

Reply:
@domi4484 , yep, same logic as in QGIS, where there it shows a QGIS logo.